source/brailleTables.py

```diff
@@ -29,6 +30,13 @@ class TableSource(StrEnum):
 	"""The name of the scratchpad table source"""
 
 
+class TableType(StrEnum):
+	INPUT = "input"
+	"""The name of the input type for braille tables"""
+	OUTPUT = "OUTPUT"
+	"""The name of the output type for braille tables"""
+
```

Comment:
I don't think this needs to be a StrEnum, what are the strings being used for? 
Otherwise, the string casings should match

```suggestion
class TableType(Enum):
	INPUT = auto()
	"""Input type for braille tables for back-translation"""
	OUTPUT = auto()
	"""Output type for braille tables for translation"""

```